components/Companies.tsx

```diff
@@ -72,3 +110,32 @@ const Tag = ({ children }) => {
     </div>
   );
 };
+
+const SelectTags = ({ onChange }) => {
+  return (
+    <select
+      className="text-black my-3 rounded-lg"
+      name="tags"
+      id="tags"
+      onChange={onChange}
+    >
+      <option value="">--Select Tag--</option>
+      <option value="Websites">Websites</option>
+      <option value="Design">Design</option>
+      <option value="Network">Network</option>
+      <option value="SEO">SEO</option>
+      <option value="Marketing">Marketing</option>
+      <option value="Consulting">Consulting</option>
+      <option value="Software">Software</option>
+      <option value="Financial">Financial</option>
+    </select>
+  );
+};
+
+const NoResult = () => {
+  return (
+    <div className="col-span-full p-5">
+      <h4 className="text-center text-2xl">No Results Found</h4>
```

Comment:
The No Results Found text probably shouldn't be a h4. Let's just use styles instead.